### PR TITLE
Dutch translation

### PR DESCRIPTION
**Pull request by ProkhorZ**
Update of the NL translation for release 0.12.1. according to issue #1377. Paging @erciccione.

### CODE REVIEW

File: translations/monero-core_nl.ts [hunk 1]
```diff
@@ -580,217 +553,212 @@ S Send, R Receive, H History, D Advanced, E Settings.</translatorcomment>
...
 </context>
 <context>
     <name>ProgressBar</name>
     <message>
-        <location filename="../components/ProgressBar.qml" line="41"/>
-        <source>Establishing connection...</source>
-        <translation>Bezig met verbinding te maken...</translation>
+        <location filename="../components/ProgressBar.qml" line="38"/>
+        <source>%1 blocks remaining: </source>
+        <translation>Blokken te gaan in %1: </translation>
```
Comment (skftn) [inline translations/monero-core_nl.ts]: Wellicht: `%1 Blokken te gaan:`

File: translations/monero-core_nl.ts [hunk 2]
```diff
@@ -1897,22 +2073,22 @@ Voor een betalingsbewijs hoeft u het adres van de ontvanger niet op te geven.</t
...
         <source>(optional)</source>
         <translation>(optioneel)</translation>
     </message>
     <message>
-        <location filename="../wizard/WizardDaemonSettings.qml" line="170"/>
-        <source>Connect to a remote node until my own node has finished syncing</source>
-        <translation>Verbinding maken met een externe node totdat mijn eigen node is gesynchroniseerd</translation>
+        <location filename="../wizard/WizardDaemonSettings.qml" line="183"/>
+        <source>Bootstrap node (leave blank if not wanted)</source>
+        <translation>Externe node (leeg laten als u die niet gebruikt)</translation>
```
Comment (skftn) [inline translations/monero-core_nl.ts]: Wellicht handig om deze wel `Bootstrap` te laten, gezien dit in de GUI een andere functionaliteit heeft dan puur het gebruik van een externe node.

File: translations/monero-core_nl.ts [hunk 3]
```diff
@@ -1104,330 +1068,511 @@ De naam van het oude cachebestand wordt gewijzigd, zodat het later kan worden he
...
-        <location filename="../pages/Settings.qml" line="215"/>
+        <location filename="../pages/Settings.qml" line="226"/>
         <source>Remote Node</source>
         <translation>Externe node</translation>
     </message>
     <message>
-        <location filename="../pages/Settings.qml" line="233"/>
+        <location filename="../pages/Settings.qml" line="240"/>
+        <source>Bootstrap node</source>
+        <translation>Externe node</translation>
```
Comment (skftn) [inline translations/monero-core_nl.ts]: Wellicht handig om deze wel `Bootstrap` te laten, gezien dit in de GUI een andere functionaliteit heeft dan puur het gebruik van een externe node.

File: translations/monero-core_nl.ts (hunk 1 above)
Comment (ProkhorZ) [reply translations/monero-core_nl.ts]: That's what I thought at first, but if you look at the ProgressBar code, you'll see that the placeholders %1 stands for the syncType: Wallet/Daemon. 
```
    property string syncType // Wallet or Daemon
    property string syncText: qsTr("%1 blocks remaining: ").arg(syncType)
```
The number of remaining blocks is in the var `remaining` IIUC.

File: translations/monero-core_nl.ts (hunk 2 above)
Comment (skftn) [reply translations/monero-core_nl.ts]: You'd look 'bootstrap' up, at which point you'll more easily find documentation regarding bootstrap nodes, as opposed to translating 'externe node', which yields totally different documentation. I'd also argue that there are many english words in the dutch language.

Your choice though.

Comment (ProkhorZ) [reply translations/monero-core_nl.ts]: Good point! I will change it. I left *key image* untranslated for the same reason.

**Comment by ProkhorZ** [comment]
Don't worry, I'll squash those commits later...

**Comment by ProkhorZ** [comment]
@erciccione From where should I run those commands? Starting from zero, I did the following:

```
git clone https://github.com/ProkhorZ/monero-gui
cd monero-gui
git checkout dutch-0.12.1
```

Is that correct?

**Comment by erciccione** [comment]
It is, now you need to se up the remote upstream:

`git remote add upstream https://github.com/monero-project/monero-gui.git`

after that, you can use the commands i wrote earlier. You will get a warn after `git pull upstream master --rebase`, but that's ok

**Comment by ProkhorZ** [comment]
OK, rebased.

**Comment by ProkhorZ** [comment]
... and commits squashed.

**Pull request merged**